tests/js/client/shell/http-fuzzer.js

```diff
@@ -0,0 +1,126 @@
+/*jshint globalstrict:false, strict:false, maxlen: 500 */
+/*global assertTrue, assertEqual, arango*/
+
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Julia Puget
+////////////////////////////////////////////////////////////////////////////////
+
+const jsunity = require("jsunity");
+const internal = require('internal');
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+function httpRequestsFuzzerTestSuite() {
+  return {
+    setUpAll: function () {
+    },
+
+    tearDownAll: function () {
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief CoordinatorMetricsTestSuite tests
+////////////////////////////////////////////////////////////////////////////////
+    testRandReqs: function () {
+      for (let i = 0; i < 15; ++i) {
```

Comment:
Not sure about the right values here. Something that runs <= 3 minutes in total...